token/src/instruction.rs

```diff
@@ -26,23 +31,33 @@ pub enum TokenInstruction {
     ///
     /// # Accounts expected by this instruction:
     ///
-    ///   0. `[writable, signer]` New mint to create.
+    ///   0. `[writable, signer]` Mint account to initialize
     ///   1.
     ///      * If supply is non-zero: `[writable]` Account to hold all the newly minted tokens.
     ///      * If supply is zero: `[]` Owner of the mint.
-    ///   2. Optional: `[]` Owner of the mint if supply is non-zero, if present then the 
+    ///   2. Optional: `[]` Owner of the mint if supply is non-zero, if present then the
     ///      token allows further minting of tokens.
     InitializeMint(TokenInfo),
     /// Initializes a new account.  The new account can either hold tokens or be a delegate
     /// for another account.
     ///
     /// # Accounts expected by this instruction:
     ///
-    ///   0. `[writable, signer]`  New account being created.
+    ///   0. `[writable, signer]`  Account to initialize
     ///   1. `[]` Owner of the new account.
     ///   2. `[]` Token this account will be associated with.
     ///   3. Optional: `[]` Source account that this account will be a delegate for.
     InitializeAccount,
+    /// Initializes a multisignature account with N provided signers.
+    /// Multisignature accounts can take the place of an "Owner" account in any token instructions that
+    /// require an owner to be a signer.  The variant field represents the number of required
+    /// signers (M).
+    ///
+    /// # Accounts expected by this instruction:
+    ///
+    ///   0. `[signer]` Multisignature account to initialize
```

Comment:
I don't think we need this account to sign (InitializeAccount/Mint eventually too).  Same logic as https://github.com/solana-labs/solana-program-library/pull/67#discussion_r447915150.  usecase:  if the account is a derived address, that address literally can't be a signer

Reply:
Yeah, was thinking about that and planning to address that together with the other instructions as a separate change